### PR TITLE
Add versions + jwxml to install_requires in setup.py

### PR DESCRIPTION
**Pull request by josePhoenix**
Once the `.travis.yml` changes land, we'll need to put `jwxml>=0.1.0` in the `PIP_DEPENDENCIES` variable too for consistency.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -112,8 +112,14 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['numpy', 'matplotlib', 'scipy', 'poppy', 'astropy'],
-      setup_requires=['numpy', 'astropy'],
+      install_requires=[
+          'numpy>=1.10.0',
+          'matplotlib>=1.5.3',
```
Comment (mperrin) [inline setup.py]: Why are you pushing these versions to the most recent? I predict there are a LOT of users out there who don't have the most recent matplotlib that was released just a few weeks ago. I know it fixed that bug you were talking about, but is it strictly necessary as an "install requires"?

Comment (josePhoenix) [inline setup.py]: We can require something older, but unless we're adding another dimension to the build matrix we're only testing on the latest. Not specifying a version at all leads to surprises.

Comment (mperrin) [inline setup.py]: Agreed we should only test on the latest matplotlib. Couldn't we set it up to test on the latest but have the install requires set to allow anything 1.5.0 or newer?  Maybe anaconda & astroconda is increasingly smoothing the installation process enough that I'm over thinking things - but from past experience I predict headaches from some astronomers if trying to install webbpsf requires updating a whole bunch of the core scipy stack.   I dunno. 

Perhaps @jhunkeler can advise on best practices here?

Comment (josePhoenix) [inline setup.py]: Leaving off the final / patch version number makes sense. I can go ahead and make that change.

I've been conditioned to trust the automatic dependency resolver :) I understand that enough of our colleagues have been burned by buggy software releases that they exercise an abundance of caution in upgrading. Nevertheless, user support wise, it's much easier to help someone through an "unsupported version" issue than a "something breaking in a subtle way in an untested configuration" issue.

**Pull request merged**